src/components/popup.tsx

```diff
@@ -28,46 +30,38 @@ const Popup = () => {
       }
 
       setActiveColor(result);
-      setShowCustomInput(!colors.includes(result));
     }
 
     getActiveColorEffect();
   }, []);
 
   return (
     <div className="colorLinks">
-      <span
+      <h1
         style={{
           color: activeColor,
-          paddingBottom: '5px',
-          transition: 'color .5s',
-        }}
+      }}
       >
-        A Quick Brown Fox Jumped
-      </span>
+        <img aria-hidden="true" src="../links48.png" />
+        <span>color links</span>
+      </h1>
       <div className="colorLinks--grid">
-        {colors.map((color) => (
+        {COLORS.map((color) => (
           <ColorButton
             color={color}
             clickHandler={() => onColorChange(color)}
             key={color}
             active={color === activeColor}
           />
         ))}
-
-        <div>
-          <button
-            type="button"
-            aria-label="custom color"
-            className={`colorLinks--button${!colors.includes(activeColor) ? ' active' : ''}`}
-            onClick={() => setShowCustomInput(true)}
-          >
-            #
-          </button>
-        </div>
       </div>
 
-      {showCustomInput && <CustomInput color={activeColor} saveHandler={onColorChange} />}
+      <details>
```

Comment:
Still need to refactor (remove?) the hook logic, since the sweet new `<details>` element makes this easy.

Reply:
Oh that is 🔥